src/runtime/pipeline/pipeline_struct.h

```diff
@@ -367,11 +502,21 @@ struct ParamConnectionConfig {
  *\brief Backend Runtime.
  */
 class BackendRuntime {
+  using ModuleInputPairList = std::vector<std::pair<std::shared_ptr<BackendRuntime>, int>>;
+
  private:
   /*\brief The index of runtime indicates the runtime position in the pipeline.*/
   int runtime_idx_;
   /*\brief The Runtime module of a backend graph executor.*/
   Module module_;
+  /*\brief The thread is associated with the current runtime*/
+  std::thread thread_;
+  /*\brief A list of runtime which depends on the current runtime.*/
+  std::unordered_map<int, ModuleInputPairList> children_;
+  /*\brief A map including the runtime input index and the notification data structure.*/
+  std::unordered_map<int, std::shared_ptr<DataNotify>> parents_notify_;
+  /*\brief The times of using pipeline function. */
+  uint32_t statistic_pipeline_execute_times_ = 0;
```

Comment:
This variable names and its doc does not make a lot of sense ("the times of ..." is not really a word). In the next PR please improve them.

Reply:
sure, will do that, thanks @masahi.